backend/src/relation-requests/relation-requests.resolver.spec.ts

```diff
@@ -1,19 +1,368 @@
+      const data: RelationRequestsInput = {
+        userSenderId: '1',
+        userReceiverId: '2'
+      }
+      const resExpected = { id: '1', ...data }
+      relationRequestsService.create.mockReturnValue(resExpected)
+
+      const result = await relationRequestResolver.createRelationRequests(data)
+
+      expect(result).toStrictEqual(resExpected)
+      expect(relationRequestsService.create).toHaveBeenCalledWith(data)
+    })
+
+    it('deleteRelationRequests', async () => {
+      const resExpected = { id: '1' }
+      relationRequestsService.delete.mockReturnValue(resExpected)
+
+      const result = await relationRequestResolver.deleteRelationRequests(
+        '1',
+        '2'
+      )
+      expect(result).toStrictEqual(resExpected)
+      expect(relationRequestsService.delete).toHaveBeenCalledWith('1', '2')
+    })
+  })
+  describe('Test Query', () => {
+    it('findOneRelationRequests', async () => {
+      const resExpected = {
+        userSenderId: '1',
+        userReceiverId: '2'
+      }
+      relationRequestsService.findOne.mockReturnValue(resExpected)
+
+      const result = await relationRequestResolver.findOneRelationRequests(
+        '1',
+        '2'
+      )
+
+      expect(result).toStrictEqual(resExpected)
+      expect(relationRequestsService.findOne).toHaveBeenCalledWith('1', '2')
+    })
+
+    it('isRelationRequestsRequested', async () => {
+      const resExpected = {
+        userSenderId: '1',
+        userReceiverId: '2'
+      }
+      relationRequestsService.isRequested.mockReturnValue(resExpected)
+
+      const result = await relationRequestResolver.isRelationRequestsRequested(
+        '1',
+        '2'
+      )
+
+      expect(result).toStrictEqual(resExpected)
+      expect(relationRequestsService.isRequested).toHaveBeenCalledWith('1', '2')
+    })
+    it('findAllRelationRequestsReceived', async () => {
+      const resExpected = {
+        userSenderId: '1',
+        userReceiverId: '2'
+      }
+      relationRequestsService.findAllRequestReceived.mockReturnValue(
+        resExpected
+      )
+
+      const result =
+        await relationRequestResolver.findAllRelationRequestsReceived('1')
+
+      expect(result).toStrictEqual(resExpected)
+      expect(
+        relationRequestsService.findAllRequestReceived
+      ).toHaveBeenCalledWith('1')
+    })
+    it('findAllRelationRequestsSent', async () => {
+      const resExpected = {
+        userSenderId: '1',
+        userReceiverId: '2'
+      }
+      relationRequestsService.findAllRequestSent.mockReturnValue(resExpected)
+
+      const result = await relationRequestResolver.findAllRelationRequestsSent(
+        '1'
+      )
+
+      expect(result).toStrictEqual(resExpected)
+      expect(relationRequestsService.findAllRequestSent).toHaveBeenCalledWith(
+        '1'
+      )
+    })
   })
 
   it('should be defined', () => {
-    expect(resolver).toBeDefined()
+    expect(relationRequestResolver).toBeDefined()
+  })
```

Comment:
Between Test Query and Test Error:
Add 2 tests like in user resolver spec, one to check that everything work in createUser and one for updateUser, all in a ValidationPipe describe:
```ts
describe('Test ValidationPipe', () => {
    it('createUser', async () => {
      const data = {
        mail: 'dhaya@gmail.com',
        username: 'dhaya',
        avatarUrl: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ',
        status: EStatus.DoNotDisturb,
        languages: ELanguage.English,
        level: 3
      }
      const metadata: ArgumentMetadata = {
        type: 'body',
        metatype: CreateUserInput,
        data: ''
      }
      const response = await validationPipe.transform(data, metadata)
      expect(response).toStrictEqual(data)
    })
    it('updateUser', async () => {
      const data = {
        username: 'dhaya',
        avatarUrl: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ',
        status: EStatus.DoNotDisturb,
        languages: ELanguage.English,
        level: 3
      }
      const metadata: ArgumentMetadata = {
        type: 'body',
        metatype: UpdateUserInput,
        data: ''
      }
      const response = await validationPipe.transform(data, metadata)
      expect(response).toStrictEqual(data)
    })
  })
  ```